EventDisplay/include/EventDisplay/EveShapeDrawer.h

```diff
@@ -16,7 +16,7 @@
 #include "TGeoMatrix.h"
 #include "TGeoShape.h"
 #include "TGeoTube.h"
-#include "TVector3.h"
+#include "TVector3D.h"
```

Comment:
I think this missed the Math, right?

Reply:
This isn't even compiled. Its the old-old TEve-based event display that I abandoned when developing the v12 geometry. We could delete these files to avoid confusion and direct people to use and contribute to https://github.com/LDMX-Software/event-vis

Reply:
So I think the `EventDisplay` directory should instead be the processors that create the JSON that could be fed into the pheonix-based event-vis repo linked above. There's also some other Python/Matplotlib vis tools that are in the `EventDisplay/python` directory that I think could be kept around.